plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/TransformationMatrix.cpp

```diff
@@ -1261,7 +1317,14 @@ MStatus TransformationMatrix::translateTo(const MVector& vector, MSpace::Space s
     {
       insertTranslateOp();
     }
-    pushTranslateToPrim();
+
+    // Push new value to prim, but only if it's changing, otherwise extra work and unintended
+    // side effects will happen.
+    //
+    if (!vector.isEquivalent(m_translationFromUsd))
+    {
+      pushTranslateToPrim();
+    }
```

Comment:
So, this is a follow-up to [this thread](https://github.com/Autodesk/maya-usd/pull/183#discussion_r377780255), because I can't find a way to add more replies to an "outdated" review...

Anyway - I was mostly advocating zeroing the tweak because that's what the existing code would do, if pushTranslateToPrim were called. However - you're totally right, I now realize that this extra step is unnecessary, since [this line](https://github.com/Autodesk/maya-usd/blob/37bf63327f18c7944e30f715d9f061b4338e5c75/plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/TransformationMatrix.cpp#L1305)  is already zeroing tweak, if vector == m_translationFromUsd!